### ISSUE TITLE
About chinese text recognition

### ISSUE THREAD

**Issue by isyyzhang** [ayumiymk/aster.pytorch#71]
Thanks for your share, but have you ever tried trainning with chinese text?

**Comment by ayumiymk** [comment]
Yes, with minor modifications, I tried Chinese text recognition based on this repo. You can try it.

**Comment by isyyzhang** [comment]
> Yes, with minor modifications, I tried Chinese text recognition based on this repo. You can try it.

Thanks for your reply, I have changed the vocabulary into a Chinese character set and tried trainning with 2 million images. Besides, the input size of my dataset is (32, 320) and I closed the STN module. My loss finally stayed around 3.2 and couldn't go down any further. Is there anything I ignored?
